### DISCUSSION TITLE
Airspy R2 Dropped Samples?

### DISCUSSION THREAD

**Discussion by pmaine** [AlexandreRouma/SDRPlusPlus#308] [Q&A] [answered]
I over clocked my Raspberry Pi to 2Ghz and I also have SATA SD card connected to my Pi. When I start up SDR++ with my Airspy and sample set to 10Mhz , I hear sputtering audio which I attribute to the Pi dropping samples. If I drop the packet size down to 2.5 Mhz, it works fine.

Last night I tried the same experiment with a high end intel processor laptop running Ununtu. SDR++ work fine when set to 10Mhz. By that I mean there was no sputtering audio.

Is there a way to determine if I am loosing samples on the high end Linux/Intel laptop?

Thank you

**Comment by AlexandreRouma** [comment]
If you're losing samples it'll be obvious because you'll get audio glitches.

No amount of overclocking will let the pi run the a GUI SDR app at more than 3 or 4 MS/s

**Comment by starvald** [comment, marked as answer]
even with a bash program like rtl_airband 10MHz is too much for the Pi4

from https://airspy.com/quickstart/
**Troubleshooting PC performance problems**

run airspy_rx -r NUL -t 0
If the average throughput is below 10.0 MSPS then either your USB controller has problems or you CPU can’t process the data.

but also factor in the overhead needed to run sdr++
